### PR TITLE
Fixes SeIncreaseQuotaPrivilege

### LINKED ISSUE #20: The 'should include' does not check for unwanted accounts
The 'should include' doesn't check that unwanted accounts are not present so this control is invalid.

control 'cis-adjust-memory-quotas-2.2.5' do
  impact 0.7
  title '2.2.5 Set Adust memory quotas for a process to Administrators, LOCAL SERVICE, NETWORK SERVICE'
  desc 'Set Adust memory quotas for a process to Administrators, LOCAL SERVICE, NETWORK SERVICE'
  describe security_policy do
    its('SeIncreaseQuotaPrivilege') { should include 'S-1-5-19' }
    its('SeIncreaseQuotaPrivilege') { should include 'S-1-5-20' }
    its('SeIncreaseQuotaPrivilege') { should include 'S-1-5-32-544' }
  end
end

I suggest the following. This will work only if the returned array is always sorted which I think is true.
Otherwise we need to check no other account is added.

control 'cis-adjust-memory-quotas-2.2.5' do
  impact 0.7
  title '2.2.5 Set Adust memory quotas for a process to Administrators, LOCAL SERVICE, NETWORK SERVICE'
  desc 'Set Adust memory quotas for a process to Administrators, LOCAL SERVICE, NETWORK SERVICE'
  describe security_policy do
   its('SeIncreaseQuotaPrivilege') { should eq ['S-1-5-19','S-1-5-20','S-1-5-32-544'] }
  end
end

### PR DESCRIPTION
**Pull request by imjoseangel**
Signed-off-by: imjoseangel <josea.munoz@gmail.com>

Fixes #20 and #22

### CODE REVIEW

File: controls/local_policies.rb [hunk 1]
```diff
@@ -104,9 +104,7 @@
   ref 'Umsetzungshinweise zum Baustein SYS.1.2.2: Windows Server 2012', url: 'https://www.bsi.bund.de/SharedDocs/Downloads/DE/BSI/Grundschutz/IT-Grundschutz-Modernisierung/UH_Windows_Server_2012.html'
   ref 'Center for Internet Security', url: 'https://www.cisecurity.org/'
   describe security_policy do
-    its('SeIncreaseQuotaPrivilege') { should include 'S-1-5-19' }
-    its('SeIncreaseQuotaPrivilege') { should include 'S-1-5-20' }
-    its('SeIncreaseQuotaPrivilege') { should include 'S-1-5-32-544' }
+    its('SeIncreaseQuotaPrivilege') { should eq ['S-1-5-19', 'S-1-5-20', 'S-1-5-32-544'] }
```
Comment (rndmh3ro) [inline controls/local_policies.rb:107]: Does this still work, if it is not in this exact order?

Comment (imjoseangel) [reply controls/local_policies.rb:107]: Hi @rndmh3ro ,

I have tested it and is about how Windows manages the list:

By default you have:

```
Administrators
LOCAL SERVICE
NETWORK SERVICE
```

If you remove Administrators group, you get:

```ruby
  ×  windows-014: Ensure 'Adjust memory quotas for a process' is set to 'Administrators, LOCAL SERVICE, NETWORK SERVICE'
     ×  Security Policy SeIncreaseQuotaPrivilege is expected to eq ["S-1-5-19", "S-1-5-20", "S-1-5-32-544"]
     
     expected: ["S-1-5-19", "S-1-5-20", "S-1-5-32-544"]
          got: ["S-1-5-19", "S-1-5-20"]
     
     (compared using ==)
```

When adding back, Windows returns the list in that exact order always:

```rb
  ✔  windows-014: Ensure 'Adjust memory quotas for a process' is set to 'Administrators, LOCAL SERVICE, NETWORK SERVICE'
     ✔  Security Policy SeIncreaseQuotaPrivilege is expected to eq ["S-1-5-19", "S-1-5-20", "S-1-5-32-544"]
```

If you add an extra user:

```rb
  ×  windows-014: Ensure 'Adjust memory quotas for a process' is set to 'Administrators, LOCAL SERVICE, NETWORK SERVICE'
     ×  Security Policy SeIncreaseQuotaPrivilege is expected to eq ["S-1-5-19", "S-1-5-20", "S-1-5-32-544"]
     
     expected: ["S-1-5-19", "S-1-5-20", "S-1-5-32-544"]
          got: ["S-1-5-19", "S-1-5-20", "imjoseangel", "S-1-5-32-544"]
     
     (compared using ==)
```

etc

Comment (rndmh3ro) [reply controls/local_policies.rb:107]: Thanks for investigating this!

**Comment by chris-rock** [comment]
Thank you @imjoseangel for this improvement!

**Pull request merged**